bundles/org.openhab.binding.siemenshvac/src/main/java/org/openhab/binding/siemenshvac/internal/Metadata/SiemensHvacMetadataDataPoint.java

```diff
@@ -0,0 +1,241 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.siemenshvac.internal.Metadata;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.siemenshvac.internal.constants.SiemensHvacBindingConstants;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+
+/**
+ *
+ * @author Laurent Arnal - Initial contribution
+ */
+@NonNullByDefault
+public class SiemensHvacMetadataDataPoint extends SiemensHvacMetadata {
+    private String dptId = "-1";
+    private String dptType = "";
+    private @Nullable String dptUnit = null;
+    private @Nullable String min = null;
+    private @Nullable String max = null;
+    private @Nullable String resolution = null;
+    private @Nullable String fieldWitdh = null;
+    private @Nullable String decimalDigits = null;
+    private Boolean detailsResolved = false;
+    @SuppressWarnings("unused")
+    private Boolean unresolvableDetails = false;
+    private @Nullable String dialogType = null;
+    private @Nullable String maxLength = null;
+    private @Nullable String address = null;
+    private int dptSubKey = -1;
+    private boolean writeAccess = false;
+    private List<SiemensHvacMetadataPointChild> child;
```

Comment:
Initialization with `List.of()` would avoid it to be `@Nullable`

Reply:
Not sure of the syntax, do you want to say :
private List<SiemensHvacMetadataPointChild> child = List.of();